# w1 is 0.25
# w2 is 0.3
# h1 is 0.265
# h2 is 0.23
# size is [width,height,width_offset,height_offset]
bin_A : [0.25,0.265,0,0]
bin_C : [0.25,0.265,0,0]
bin_J : [0.25,0.265,0,0]
bin_L : [0.25,0.265,0,0]

bin_D : [0.2265, 0.18, -0.004, 0.022]
bin_F : [0.25,0.23,0,0]
bin_G : [0.25,0.23,0,0]
bin_I : [0.25,0.23,0,0]

bin_B : [0.3,0.265,0,0]
bin_K : [0.3,0.265,0,0]

bin_E : [0.3,0.23,0,0]
bin_H : [0.3,0.23,0,0]
